app/src/components/SoldArtworkList.tsx

```diff
@@ -44,15 +44,17 @@ class SoldArtworkList extends React.Component<ContractProps, SoldArtworkListStat
       'RecordARR', options).then(function (events: any) {
       eventsFound = events.length;
```

Comment:
wont this mean that eventsFound's final value after the loop will be events.length + (number of events with right returnValues.from)?

Reply:
You are correct, will fix